src/BugsnagUnity/Native/Cocoa/Configuration.cs

```diff
@@ -10,10 +10,11 @@ class Configuration : AbstractConfiguration
   {
     internal IntPtr NativeConfiguration { get; }
 
-    internal Configuration(string apiKey) : base()
+    internal Configuration(string apiKey, bool autoNotify) : base()
```

Comment:
Same as Android, this implementation of `Configuration` does not override `AutoNotify` so it will also need to be set from the `autoNotify` parameter. Otherwise Unity log messages could still generate reports.